modules/osdk-suggested-namespace-node-selector.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-suggested-namespace-default-node_{context}"]
+= Setting a suggested namespace with default node selector
+
+Some Operators expect to run only on primary nodes, which can be done by setting a `nodeSelector` in the Pod spec by the Operator itself.
+
+To avoid getting a duplicated and potential conflicting cluster-wide default `nodeSelector`, you can set a default node selector on the namespace where the Operator runs. The default node selector will take precedence over the cluster default so the cluster default will not be applied to the pods in the Operators namespace.
+
+When adding the Operator to a cluster using OperatorHub, the web console auto-populates the suggested namespace for the cluster administrator during the installation process, and the suggested namespace is created directly using the namespace manifest in YAML included in the cluster service version (CSV).
```

Comment:
Should there be an 'article' or something else preceding '...manifest in YAML included...' to make that part of the sentence clearer?